scripts/exregional_refl2tten.sh

```diff
@@ -182,12 +182,17 @@ fi
 process_radarref_path=${CYCLE_DIR}/process_radarref
 process_lightning_path=${CYCLE_DIR}/process_lightning
 
-obs_file=${process_radarref_path}/RefInGSI3D.dat
-if [ -r "${obs_file}" ]; then
-   cp_vrfy "${obs_file}" "RefInGSI3D.dat_01"
-else
-   print_info_msg "$VERBOSE" "Warning: ${obs_file} does not exist!"
-fi
+ss=0
+for bigmin in ${RADARREFL_TIMELEVEL}; do
```

Comment:
change to 
```
for bigmin in ${RADARREFL_TIMELEVEL[@]}; do
   bigmin=$( printf %2.2i $bigmin )
```
to match changes in ush/config_defaults.sh

Reply:
Done